src/mlpack/methods/decision_tree/all_categorical_split.hpp

```diff
@@ -34,58 +36,59 @@ class AllCategoricalSplit
   /**
    * Check if we can split a node.  If we can split a node in a way that
    * improves on 'bestGain', then we return the improved gain.  Otherwise we
-   * return the value 'bestGain'.  If a split is made, then classProbabilities
-   * and aux may be modified.  For this particular split type, aux will be empty
-   * and classProbabilities will hold one element---the number of children.
+   * return the value 'bestGain'.  If a split is made, then splitInfo and
+   * aux may be modified.  For this particular split type, aux will be empty
+   * and splitInfo will store the number of children of the node.
    *
    * @param bestGain Best gain seen so far (we'll only split if we find gain
    *      better than this).
    * @param data The dimension of data points to check for a split in.
    * @param numCategories Number of categories in the categorical data.
    * @param labels Labels for each point.
+   * @param begin Start index of labels.
```

Comment:
I'd really prefer to avoid adding this parameter, if possible.

Reply:
I also didn't want it. But I was getting some errors (see the other comment) which forced me to use this :(